diff --git a/target/linux/lantiq/files-4.14/arch/mips/boot/dts/ARV752DPW22.dts b/target/linux/lantiq/files-4.14/arch/mips/boot/dts/ARV752DPW22.dts
index 1d4d662ab3..cf263ecef9 100644
--- a/target/linux/lantiq/files-4.14/arch/mips/boot/dts/ARV752DPW22.dts
+++ b/target/linux/lantiq/files-4.14/arch/mips/boot/dts/ARV752DPW22.dts
@@ -9,7 +9,7 @@
 	model = "Arcor 803";
 
 	chosen {
-		bootargs = "console=ttyLTQ0,115200";
+		bootargs = "console=ttyLTQ0,115200 cca=3";
 	};
 
 	aliases {
